contents/handbook/company/communication.md

```diff
@@ -36,23 +36,30 @@ We make things public by default because [transparency](/handbook/company/cultur
 Information that is not publicly shared is in areas with complex signals that can impact our ability to sell, raise money or are inappropriate to share more widely for personal privacy reasons.
 
 
-We have two repos to centralize and document all internal communication. These are the source of truth for any internal information, and anything that should be written down (as established in these guidelines) should live here, not on Slack. This will make it easier when having to search for older stuff, sharing context between public and internal repos, and for newcomers to have all information they might need readily available.
+We have two repos to centralize and document private internal communication. These are the source of truth for any internal information, and anything that should be written down (as established in these guidelines) should live in these repos or (better) in this Handbook, not on Slack. This will make it easier when having to search for older stuff, sharing context between public and internal repos, and for newcomers to have all information they might need readily available.
 
 ### Company Internal
 Repository can be found in https://github.com/PostHog/company-internal
 
-Documents any company-wide internal information, in addition to any information related to People, Ops, Legal & Compliance, Finance or Strategy.
+Documents any company-wide information that can't be share publicly within People, Ops, Legal, Finance or Strategy.
 
 **Examples of information that should go here:**
-- ✅ Hiring plans and discussions before we post a job ad
+- ✅ Hiring plans and discussions _before_ we post a job ad
 - ✅ People discussions, e.g. benefits, pensions, share options, org structure
 - ✅ Onboarding/offboarding checklists
-- ✅ Non-engineering team sprint planning
-- ✅ Sensitive discussions about future positioning, customer strategy, fundraising, board meetings
-- ✅ Discussions about replacing or adding tools, services, and systems that we use
+- ✅ Non-engineering team sprint planning (as these will often be a mix of public and private tasks and we don't want to restrict people)
+- ✅ [Sometimes] Sensitive discussions about future positioning, customer strategy, fundraising, board meetings
+- ✅ [Sometimes] Discussions about replacing or adding tools, services, and systems that we use
```

Comment:
Why does this need to be private - could it get us in legal trouble?

Reply:
Re tools - it's more a security risk. I wouldn't want to have a public conversation about changing bank in case it results in phishing etc., but it would be fine to talk about changing support tool publicly.